src/components/Overlay/Overlay.scss

```diff
@@ -27,3 +27,7 @@
 .ms-Overlay--dark {
   background-color: $ms-color-blackTranslucent40;
 }
+
+.ms-noScroll {
```

Comment:
This should probably be a utility class (`.ms-u-noScroll` or `.ms-u-overflowHidden`), although that would mean moving it over to Fabric Core. Or should we have component-specific utilities? Honestly it's probably not worth the trouble right now, maybe just rename the class and leave it in this file.